catalog/files/impl/src/main/java/org/projectnessie/catalog/files/adls/AdlsProgrammaticOptions.java

```diff
@@ -15,117 +15,40 @@
  */
 package org.projectnessie.catalog.files.adls;
 
-import com.google.errorprone.annotations.CanIgnoreReturnValue;
-import java.time.Duration;
+import java.util.LinkedHashMap;
 import java.util.Map;
 import java.util.Optional;
-import java.util.OptionalInt;
-import java.util.OptionalLong;
 import org.immutables.value.Value;
-import org.projectnessie.catalog.secrets.BasicCredentials;
-import org.projectnessie.catalog.secrets.KeySecret;
 
 @Value.Immutable
-public interface AdlsProgrammaticOptions extends AdlsOptions<AdlsFileSystemOptions> {
+@SuppressWarnings("immutables:from")
+public interface AdlsProgrammaticOptions extends AdlsOptions {
   @Override
   Optional<AdlsFileSystemOptions> defaultOptions();
 
   @Override
-  OptionalLong writeBlockSize();
-
-  @Override
-  OptionalInt readBlockSize();
-
-  @Override
-  Map<String, String> configurationOptions();
-
-  @Override
-  Map<String, AdlsFileSystemOptions> fileSystems();
-
-  static Builder builder() {
```

Comment:
I hesitated a bit but I think it's best to remove the builder interfaces as they were slowly getting out of sync with the upper interface (missing methods).